.github/workflows/build_prs.yaml

```diff
@@ -20,3 +20,4 @@ jobs:
       - run: yarn install --frozen-lockfile --ignore-scripts
       - run: yarn lint
       - run: yarn build
+      #- run: yarn build:frontend # not needed
```

Comment:
We don't need this because we have https://github.com/Unleash/unleash/blob/improve-dev-build-time/.github/workflows/build_frontend_prs.yml